src/index.ts

```diff
@@ -33,7 +33,18 @@ const isDevelopmentVersion = (version: string) => {
 	return /(\.|\+|-)dev/.test(version);
 };
 
-export const compare = memoize((versionA: string, versionB: string) => {
+export const compare = memoize((versionA: string | null, versionB: string | null) => {
+	if (versionA === null && versionB === null) {
+		return 0;
+	}
+	if (versionA === null && versionB !== null) {
+		return -1;
+	}
+	if (versionA !== null && versionB === null) {
+		return 1;
+	}
+	versionA = <string>versionA;
+	versionB = <string>versionB;
```

Comment:
I've added an issue for this https://github.com/resin-io-modules/resin-semver/issues/9